proto/rill/admin/v1/api.proto

```diff
@@ -807,6 +807,7 @@ message UpdateOrganizationRequest {
   string name = 1 [(validate.rules).string.min_len = 1];
   optional string description = 2;
   optional string new_name = 3;
+  optional string new_display_name = 5;
```

Comment:
This should just be `display_name` (like `description` and `billing_email`). The only reason `new_name` has a `new_` prefix is to disambiguate it from `name`, which is used to identify the org to edit.